ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -449,6 +431,8 @@ func (r *RayJobReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 
 		// If the RayJob is completed, we should not requeue it.
 		return ctrl.Result{}, nil
+	case rayv1.JobDeploymentStatusValidationFailed:
+		// Fall through to status update
```

Comment:
I prefer change the deployment status to failed here, so `r.updateRayJobStatus(ctx, originalRayJobInstance, rayJobInstance);` can help use update the rayJobInstace (CR).
and the next reconciliation, we can remove the CR in the queue.
```suggestion
	case rayv1.JobDeploymentStatusValidationFailed:
		rayJobInstance.Status.JobDeploymentStatus = rayv1.JobDeploymentStatusFailed
```